pkg/updatestatus/nodeinformer.go

```diff
@@ -211,6 +236,73 @@ func (c *machineConfigVersionCache) match(config string) (string, bool) {
 	return v, ok
 }
 
+type machineConfigPoolSelectorCache struct {
+	cache map[string]labels.Selector
+	lock  sync.Mutex
+}
```

Comment:
consider using `sync.Map` like I suggested for MC one

Reply:
Done.

https://github.com/openshift/cluster-version-operator/pull/1164/commits/a81ec69e31f504ce10a44650cfa5c4d4260fd2cd#diff-c5a41cb83200e13544e0509a5e77c180e16a1d70d7411a79cac2bc107e89950fR212